spec/infinispan_local_spec.js

```diff
@@ -5,370 +5,274 @@ var readFile = Promise.denodeify(require('fs').readFile);
 var f = require('../lib/functional');
 var t = require('./utils/testing'); // Testing dependency
 
-describe('Infinispan local client', function() {
-  var client = t.client(t.local);
+function describeInfinispanBehaviourInLocalModeWithDifferentHotrodProtocol(options) {
+  describe('with hotrod protocol: ' + options.version, function () {
+    var client = t.client(t.local, null, options.version);
 
-  beforeEach(function(done) { client
```

Comment:
This section seems to only contain cosmetic changes: moving initial client variable to the next line. Any particular reason for doing this? In any case, if you want to make formatting changes add them as different commits.